Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/InstantiationResult.java

```diff
@@ -26,6 +35,12 @@ public Operator getOperator() {
         return operator;
     }
 
+    private Map<TypeParameter, DataType> typeMap;
+
+    public Map<TypeParameter, DataType> getTypeMap() {
+        return typeMap;
+    }
```

Comment:
The `typeMap` member or this getter aren't used anywhere yet. Should it be deleted?